apps/framework-cli/src/framework/core/infrastructure/api_endpoint.rs

```diff
@@ -159,7 +165,8 @@ impl From<EndpointFile> for ApiEndpoint {
             },
             path: value.path.clone(),
             method: Method::GET,
-            version: Version::from_string("0.0.0".to_string()),
+            // TODO: why does consumption api use patch version, but others only have major.minor?
+            version: Some(Version::from_string("0.0.0".to_string())),
```

Comment:
I believe this is dmv1 code path so it can prob stay hardcoded. But the comment still stands, I'm not sure why consumption api uses major.minor.path.

dmv2 would have to specify "0.0.0" for the diff to be the same.

Reply:
That's because that was the default here and the default in `package.json` in the default application was `0.0` is there a good reason for that?

Unfortunately not